schemas/logs/fluentd/windows_eventlog.yml

```diff
@@ -0,0 +1,136 @@
+schema: Fluentd.WindowsEventlog
+parser:
+  native:
+    name: Fluentd.WindowsEventlog
```

Comment:
Remove this. Not needed and it can cause Panther to fail. 
`native` is needed only when there is a corresponding `Go` parser in Panther.